src/main/java/baritone/pathing/movement/MovementHelper.java

```diff
@@ -295,10 +295,7 @@ static boolean canWalkOn(int x, int y, int z, IBlockState state) {
             }
             return true;
         }
-        if (block instanceof BlockStairs) {
-            return true;
-        }
-        return false;
+        return block instanceof BlockStairs;
```

Comment:
I know codacy loves this, but in the context of a chain of

```
if(cond1)
    return true
if(cond2)
    return true
if(cond3)
    return true
```

it's better if they all follow the pattern, instead of having a special case for the last one